test/integration/e2e_test.go

```diff
@@ -0,0 +1,135 @@
+package integration
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/linode/linodego"
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/oauth2"
+)
+
+const (
+	packerTemplate = "template/test_image_template.json"
+)
+
+func TestBuildPackerImage(t *testing.T) {
+	linodeToken := os.Getenv("LINODE_TOKEN")
+
+	if linodeToken == "" {
+		t.Fatal("Linode token is not set. Please set LINODE_TOKEN as environment variable.")
+	}
+
+	linodeInstanceLabel := setImageLabel()
```

Comment:
```suggestion
	linodeImageLabel := setImageLabel()
```
Might be better to name it as `linodeImageLabel`?